config/application.rb

```diff
@@ -291,9 +291,12 @@ def watchable_args
     require 'logster/redis_store'
     # Use redis for our cache
     config.cache_store = DiscourseRedis.new_redis_store
-    $redis = DiscourseRedis.new # rubocop:disable Style/GlobalVars
+    Discourse.redis = DiscourseRedis.new
     Logster.store = Logster::RedisStore.new(DiscourseRedis.new)
 
+    # Deprecated
```

Comment:
Perhaps we should just remove this in the next release so that we at least have a plan for what to do here.

Reply:
Yeah, I plan to remove it in 2.9.0.beta1 or beta2, but I need to update some plugins first.

Reply:
Sorry what I meant to say is that I think we should leave a more specific comment about when to remove this so that there is clarify for anyone else reading the codebase.